src/Events.php

```diff
@@ -140,6 +141,12 @@ protected function possibly_add_notes() {
 		InsightFirstProductAndPayment::possibly_add_note();
 		AddFirstProduct::possibly_add_note();
 		AddingAndManangingProducts::possibly_add_note();
+		CustomizingProductCatalog::possibly_add_note();
+
+		if ( $this->is_remote_inbox_notifications_enabled() ) {
+			DataSourcePoller::read_specs_from_data_sources();
+			RemoteInboxNotificationsEngine::run();
+		}
```

Comment:
I believe this isn't necessary, as it's already being done above where `possible_add_notes` is being triggered.

https://github.com/woocommerce/woocommerce-admin/pull/6032/files#diff-2e1f1cedde54c64bfd2d2d1b61019c13af57b9c5e1dc1315d4c2dcc492ad66e1R96-R99

Reply:
Thank you @louwie17 I'm not sure why that those lines are highlighted as new. The last four lines are existing code for the Remote Inbox Engine. I think we can ignore them.

Reply:
@moon0326 are you sure? they don't show up on the `main` branch: https://github.com/woocommerce/woocommerce-admin/blob/main/src/Events.php#L142

Reply:
Thanks @moon0326, approving this :)